test/fixtures/windows_beta_cluster/main.tf

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  name       = "windows-beta-cluster-${random_string.suffix.result}"
```

Comment:
In the example we should also set remove_default_node_pool to true to fully test this. Currently this is set to false by default in this example https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/a7c24d522bf5606d928babea8cc9b46c2161a5a5/examples/simple_regional_beta/variables.tf#L78-L82

Reply:
I thought about an example but at the same time this isn't a feature that is used much and it adds to an already exhaustive list of examples for GKE. I see you point about making the usage obvious through an example but I'm leaning towards not adding to an already long list especially when this is not a sought after feature.

Reply:
Agreed probably not a very used example but since we are not appending to an existing test and creating a brand new one, we should not follow the the fixtures pattern anymore.

Reply:
Added an example for this specifically